src/css/carousel.css

```diff
@@ -0,0 +1,4 @@
+.carousel {
```

Comment:
Might want to adjust the carousel size to percentage. 
This is causing the mobile version to be larger.